CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   approach to resolve an issue that could cause collection view cells to layout with 
   unexpected dimensions
 - Made new layout-based SwiftUI cell rendering option the default.
+- Fixed an issue where a UIKit view bridged to SwiftUI that wraps would always take up the proposed
```

Comment:
I guess we fixed it for AppKit too?

Reply:
TBD, waiting for some updates from @calda  since Epoxy iOS doesn't officially support native macOS, but Lottie iOS pulls in the code and uses it directly for macOS.